[2022-11-14 15:55:18,517] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: yellow_taxi_data_v2.download_dataset_task scheduled__2020-09-02T06:00:00+00:00 [queued]>
[2022-11-14 15:55:18,530] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: yellow_taxi_data_v2.download_dataset_task scheduled__2020-09-02T06:00:00+00:00 [queued]>
[2022-11-14 15:55:18,530] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-11-14 15:55:18,530] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2022-11-14 15:55:18,531] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-11-14 15:55:18,545] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): download_dataset_task> on 2020-09-02 06:00:00+00:00
[2022-11-14 15:55:18,554] {standard_task_runner.py:52} INFO - Started process 1171 to run task
[2022-11-14 15:55:18,562] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'yellow_taxi_data_v2', 'download_dataset_task', 'scheduled__2020-09-02T06:00:00+00:00', '--job-id', '224', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_gcs.py', '--cfg-path', '/tmp/tmpsqy8bg12', '--error-file', '/tmp/tmpsyksknug']
[2022-11-14 15:55:18,563] {standard_task_runner.py:77} INFO - Job 224: Subtask download_dataset_task
[2022-11-14 15:55:18,654] {logging_mixin.py:109} INFO - Running <TaskInstance: yellow_taxi_data_v2.download_dataset_task scheduled__2020-09-02T06:00:00+00:00 [running]> on host e030bae48f83
[2022-11-14 15:55:18,710] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2022-11-14 15:55:18,740] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=yellow_taxi_data_v2
AIRFLOW_CTX_TASK_ID=download_dataset_task
AIRFLOW_CTX_EXECUTION_DATE=2020-09-02T06:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2020-09-02T06:00:00+00:00
[2022-11-14 15:55:18,741] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-11-14 15:55:18,742] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'curl -sSLf https://d37ci6vzurychx.cloudfront.net/trip-data/yellow_tripdata_2020-09.parquet > /opt/***/yellow_tripdata_2020-09.parquet']
[2022-11-14 15:55:18,760] {subprocess.py:85} INFO - Output:
[2022-11-14 15:55:18,907] {subprocess.py:93} INFO - Command exited with return code 0
[2022-11-14 15:55:18,948] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=yellow_taxi_data_v2, task_id=download_dataset_task, execution_date=20200902T060000, start_date=20221114T155518, end_date=20221114T155518
[2022-11-14 15:55:19,018] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-11-14 15:55:19,075] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2022-11-16 09:23:56,789] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: yellow_taxi_data_v2.download_dataset_task scheduled__2020-09-02T06:00:00+00:00 [queued]>
[2022-11-16 09:23:56,809] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: yellow_taxi_data_v2.download_dataset_task scheduled__2020-09-02T06:00:00+00:00 [queued]>
[2022-11-16 09:23:56,809] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-11-16 09:23:56,810] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2022-11-16 09:23:56,810] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-11-16 09:23:56,846] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): download_dataset_task> on 2020-09-02 06:00:00+00:00
[2022-11-16 09:23:56,865] {standard_task_runner.py:52} INFO - Started process 567 to run task
[2022-11-16 09:23:56,873] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'yellow_taxi_data_v2', 'download_dataset_task', 'scheduled__2020-09-02T06:00:00+00:00', '--job-id', '765', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_gcs.py', '--cfg-path', '/tmp/tmppbdkvd8e', '--error-file', '/tmp/tmpdj_ot1iz']
[2022-11-16 09:23:56,875] {standard_task_runner.py:77} INFO - Job 765: Subtask download_dataset_task
[2022-11-16 09:23:56,989] {logging_mixin.py:109} INFO - Running <TaskInstance: yellow_taxi_data_v2.download_dataset_task scheduled__2020-09-02T06:00:00+00:00 [running]> on host 75ceca4d9264
[2022-11-16 09:23:57,072] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2022-11-16 09:23:57,105] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=yellow_taxi_data_v2
AIRFLOW_CTX_TASK_ID=download_dataset_task
AIRFLOW_CTX_EXECUTION_DATE=2020-09-02T06:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2020-09-02T06:00:00+00:00
[2022-11-16 09:23:57,107] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-11-16 09:23:57,108] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'curl -sSLf https://d37ci6vzurychx.cloudfront.net/trip-data/yellow_tripdata_2020-09.parquet > /opt/***/yellow_tripdata_2020-09.parquet']
[2022-11-16 09:23:57,137] {subprocess.py:85} INFO - Output:
[2022-11-16 09:23:57,505] {subprocess.py:93} INFO - Command exited with return code 0
[2022-11-16 09:23:57,548] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=yellow_taxi_data_v2, task_id=download_dataset_task, execution_date=20200902T060000, start_date=20221116T092356, end_date=20221116T092357
[2022-11-16 09:23:57,613] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-11-16 09:23:57,680] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
